package.json

```diff
@@ -109,7 +109,7 @@
     "minify": "yarn yarn-minify",
     "test": "react-app-rewired test --coverage --watchAll=false",
     "test:cypress": "yarn cypress open",
-    "test:cypress:headless": "yarn cypress run",
+    "test:cypress:headless": "yarn cypress run --browser chromium --headless",
```

Comment:
> This doesn't work on my machine (macOS), should we use `chrome` instead of `chromium` to match CI?
> 
> ```
> yarn run v1.22.18
> $ yarn cypress run --browser chromium --headless
> $ /web/node_modules/.bin/cypress run --browser chromium --headless
> Can't run because you've entered an invalid browser name.
> 
> Browser: chromium was not found on your system or is not supported by Cypress.
> 
> Cypress supports the following browsers:
>  - electron
>  - chrome
>  - chromium
>  - chrome:canary
>  - edge
>  - firefox
> 
> You can also use a custom browser: https://on.cypress.io/customize-browsers
> 
> Available browsers found on your system are:
>  - chrome
>  - electron
>  - Brave
> ```

Actually, this for for `har` thing, it crashes without it. The reason I kept it was just to make it more explicit. So we can revert it back or just keep `--headless` at least.